core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.spec.ts

```diff
@@ -120,24 +337,13 @@ describe('Opportunities List Component', () => {
     mockReloadOpportunitiesEventEmitter.emit();
     tick();
     expect(component.opportunities).toEqual(ExplorationOpportunityDict);
-    expect(component.opportunities.length).toEqual(2);
+    expect(component.opportunities.length).toEqual(16);
 
     // Removed opportunity with id as 'id2'.
     mockRemoveOpportunitiesEventEmitter.emit(['id2']);
     tick();
 
-    expect(component.opportunities.length).toEqual(1);
-    expect(component.opportunities).toEqual([
-      {
-        id: 'id1',
-        labelText: 'text',
-        labelColor: 'red',
-        progressPercentage: 50,
-        inReviewCount: 20,
-        totalCount: 100,
-        translationsCount: 50
-      }
-    ]);
+    expect(component.opportunities.length).toEqual(15);
```

Comment:
I assume there was going to be another line checking if `component.visibleOpportunities` is = `component.opportunities` right?

Reply:
Yes, we check the validity in lines 365 to 453. In this case the activePageNumber is two, with 10 opportunities per page, so the opportunities that should display are opportuntities 11 -20.

Reply:
done